src/assets/CollectiblesController.test.ts

```diff
@@ -1373,4 +1405,99 @@ describe('CollectiblesController', () => {
       });
     });
   });
+
+  describe('checkUpdateAndResetIsTransactingAndTransactionIdProps', () => {
```

Comment:
What about the case where `getCollectibleTransactionStatus` returns `undefined`?